Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_hh.gnut

```diff
@@ -0,0 +1,294 @@
+untyped
+global function GamemodeHH_Init
+
+const array<string> btmaps = [ "mp_angel_city", "mp_colony02", "mp_thaw", "mp_glitch", "mp_wargames", "mp_drydock", "mp_eden", "mp_black_water_canal", "mp_grave", "mp_complex3", "mp_homestead", "mp_forwardbase_kodai", "mp_crashsite3", "mp_rise", "mp_relic02" ]
+
+struct {
+	bool hasmadespawns = false
+	array<entity> goals
+	array<vector> spawnoffsets = [ <0,50,10>, <0,-50,10>, <50,0,10>, <-50,-0,10> ] // some sort of spawn checking (ray trace?) in the future
```

Comment:
```suggestion
	array<vector> spawnoffsets = [ < 0, 50, 10 >, < 0, -50, 10 >, < 50, 0, 10 >, < -50, -0, 10 > ] // some sort of spawn checking (ray trace?) in the future
```
its whining about this (no spaces meant it thought `<-50` was the arrow operator)